outbox/README.md

```diff
@@ -3,49 +3,75 @@
 This example demonstrates the "outbox pattern", an approach for letting services communicate in an asynchronous and reliable fashion.
 It accompanies [this post](https://debezium.io/blog/2019/02/19/reliable-microservices-data-exchange-with-the-outbox-pattern) on the Debezium blog.
 
-The sending service ("order-service") produces events in an "outbox" event table within its own local database.
+The sending service ("order-service-quarkus") produces events in an "outbox" event table within its own local database.
```

Comment:
With the old services gone, let's just omit the "-quarkus" suffix in the names.

Reply:
Do you want me to also change the directory names as well or simply the artifacts?

Reply:
With only one flavour of these services I don't see name to encode the implementation choice into any names, i.e. let's simply use "order-service" everywhere.

Reply:
Renamed as requested.